test/e2e/provision_ibmcloud.go

```diff
@@ -0,0 +1,42 @@
+//go:build ibmcloud
+// +build ibmcloud
```

Comment:
https://github.com/confidential-containers/cloud-api-adaptor/pull/597/files#r1102633220

Reply:
Yeah, it's a try to fix lint error but not worked. dropped the old one.